modules/nf-core/pairtools/sample/meta.yml

```diff
@@ -0,0 +1,44 @@
+name: "pairtools_sample"
+description: Select a random subset of pairs in a pairs file.
+keywords:
+  - sort
```

Comment:
Can you add some more keywords here?

Reply:
sorry, my fault.

Reply:
No problem :) That's what reviews are for 😁